src/pocketdb/repositories/web/WebRpcRepository.cpp

```diff
@@ -3403,24 +3400,39 @@ namespace PocketDb
                     TryBindStatementText(stmt, i++, lang);
             }
 
-            TryBindStatementInt(stmt, i++, countOut);
-
             // ---------------------------------------------
 
+            LogPrintf(sqlite3_expanded_sql(*stmt));
```

Comment:
Forgotten debug logging?

Reply:
Ahh... My fault. Forgot to remove it. Edited now.